amalgamation/amalgamation.py

```diff
@@ -149,8 +154,8 @@ def expand(x, pending, stage):
 expand.fileCount = 0
 
 # Expand the stages
-expand(sys.argv[2], [], "dmlc")
-expand(sys.argv[3], [], "nnvm")
+expand(sys.argv[2], [], "3rdparty/dmlc")
```

Comment:
what's a stage in expand()? this doesn't match the existing path. is this intended?

Reply:
The `stage` is used to match the prefix of a file. Updating it to `dmlc-core` won't affect the result. Let me document the usage.